pkg/status/controller.go

```diff
@@ -54,8 +54,8 @@ type Controller struct {
 	wdsName            string
 	wdsDynClient       dynamic.Interface
 	wdsKubeClient      kubernetes.Interface
-	imbsDynClient      dynamic.Interface
-	imbsKubeClient     kubernetes.Interface
+	ITSDynClient       dynamic.Interface
```

Comment:
In previous review initialism was intended for documentation instances only - instances in code are also acceptable in my opinion as long as they don't cause problems, such as in here where now controller has exported field members for no good reason - or consistency with other space variable namings.

So overall, I think it's consistent and best for namings in code to remain camel-case (and for documentation everywhere to use `ITS` and `its1`...)

Reply:
You convinced me in your last comment Maroon.  The term 'its' is confusing and for clarity and consistency we should stick with a single convention - ITS.  However if you feel very strongly about this, I will change it back.

Reply:
I agree with @vMaroon . In the Go language case is used to indicate public/private and should be chosen with that in mind. This overrides the norm for capitalization of initialisms. You can see many examples of this, in code in other projects as well as KubeStellar.

Reply:
Done